learn/frontend-dapp/bootstrap-dapp.md

```diff
@@ -0,0 +1,17 @@
+---
+order: 4
+---
+
+# Bootstrap dApp
+
+There are two approaches to bootstrap a new dApp: as a lerna package in the monorepo or as a standalone app.
+
+## Monorepo template
+
+With this approach, we'll create another lerna package in the `packages/` directory that will use the local `logic` and `design` packages as dependencies.
+
+For that, you only need to copy the `_template` directory into `packages/` and rename it to `balance-checker`.
```

Comment:
Can you add some links there, and code samples? They should be able to have a running dApp going by the end of the tutorial and have a clue how to hack it. Ideally, add a page on how to modify this dApp to do something.

First, something like the quick start guide:

```sh
git clone https://github.com/CosmWasm/dApps.git
cd dApps
cp -r _template packages/balance-checker
yarn && yarn build-deps
cd packages/balance-checker
yarn start
```

Once they see this, they can go onto another page that shows them how to make some simple changes to it and then view those. I don't know - Point to a new network? Query a given contract address? Something else? It doesn't need to look pretty, but should give them a rough start.

Reply:
I'll add those commands, but

> They should be able to have a running dApp going by the end of the tutorial and have a clue how to hack it

makes me think that maybe you didn't see `learn/frontend-dapp/dapp-development.md`? It was hidden by GitHub (the usual "Load large diff").

If you saw it and you'd like me to extend the example dApp with TXs I think that would be a great idea, though.

Reply:
I did not see this on the diff. Sorry

Happy if you merge this, and I will review the actual docs site. Adding sending tx would be very good